myapp/app/views/tasks/index.html.erb

```diff
@@ -8,21 +8,23 @@
       <th>Status</th>
       <th>Label</th>
       <th>Priority</th>
+      <th>Created at</th>
       <th colspan="3"></th>
     </tr>
   </thead>
 
   <tbody>
     <% @tasks.each do |task| %>
       <tr>
-        <td><%= task.name %></td>
+        <td class="task-name"><%= task.name %></td>
         <td><%= task.desc %></td>
         <td><%= task.status %></td>
```

Comment:
since you have the enum setup on the "create task" field, let's update here to show the status(and priority) with the text from enum!

![image](https://user-images.githubusercontent.com/10822260/122487143-863d7f80-d015-11eb-9643-13926beef88d.png)
![image](https://user-images.githubusercontent.com/10822260/122487154-8d648d80-d015-11eb-93ef-2ddb04e6a1b0.png)

example: 
`Task.statuses.key(task.status)`

source: 
https://stackoverflow.com/a/45374287